imgs/nwi-pr-utils/default.nix

```diff
@@ -3,15 +3,13 @@ let
   nwi = import ../../nwi.nix;
   lib = pkgs.lib;
 
-  # Locking version of tfsec to what it was in the existing GitLab CI job definitions.
-  # TODO: Upgrade to the latest and remediate all new findings.
   custom-tfsec = pkgs.tfsec.overrideAttrs (oldAttrs: rec {
```

Comment:
At this point lets just nuke the `custom-tfsec` drv and use the version thats upstream in nixpkgs

I know @davido-nw was going to us to latest relatively soon too

Reply:
Updated to remove the custom-tfsec.

This has been a really great opportunity to get my hands dirty with nix.  I feel like I'm finally getting the hang of how this is all put together finally ;)